frontend/app/services/subscription-service.server.ts

```diff
@@ -88,12 +88,11 @@ function createSubscriptionService() {
 
     const subscriptions = subscriptionsSchema.parse(await subscriptionsResponse.json())._embedded.subscriptions.map((subscription) => ({
       id: subscription.id,
-      preferredLanguage: subscription.msLanguageCode,
-      alertType: subscription.alertTypeCode,
+      msLanguageCode: subscription.msLanguageCode,
```

Comment:
I prefer that we don't change the name to `msLanguageCode` in the domain entity that this service returns. 

The API should (eventually) be returning the language ID in my opinion and the front-end application shouldn't need to know implementation details like the language code being a Microsoft language code.

```suggestion
      preferredLanguageId: subscription.msLanguageCode,
```